tests/test_vector/test_polygon.py

```diff
@@ -123,3 +124,18 @@ def test_transform_to_px_coords(self):
         output_subset = output_gdf[['geometry']].reset_index(drop=True)
 
         assert truth_subset.equals(output_subset)
+
+
+class Test_gdf_to_yolo(object):
```

Comment:
An irrelevant aside: classes are generally camelcase with no underscores (so in this case TestGdfToYolo or TestGDFToYOLO)